classes/local/renderer.php

```diff
@@ -133,7 +133,7 @@ public function get_name(): string {
      * Gets content from a template using a cache.
      * If the cache key is null, then the cache is not used.
      *
-     * @param string $cachename
+     * @param string $cachename Should either be 'cms_content' or 'cms_name'.
```

Comment:
As in it expects something to be prefixed with either `cms_content_` or `cms_name_`? or that the string itself is named `cms_content` or `cms_name`?

Reply:
The latter.

Reply:
Right I see, so we're either caching the content, or the name is the expectation here?